README.md

```diff
@@ -80,8 +80,8 @@ Concourse instance to the next.
 
 ### Learn More
 
-* The [Official Site](https://concourse-ci.org) for documentation and
-  reference material.
+* The [Official Site](https://concourse-ci.org) for documentation,
+  reference material, and example pipelines (which no longer live in this repository).
```

Comment:
Seems weird to allude to "examples" and not point people to where they live. DO we have examples that live somewhere that we can directly link to? Or is this a yet-to-be-implemented thing?

Reply:
It refers to this page: https://concourse-ci.org/examples.html 

It's in the top nav bar so I didn't link it, but can add the link if that would be better.